src/story.ts

```diff
@@ -309,7 +313,7 @@ export type StoryAnnotations<
 
   /** @deprecated */
   story?: Omit<StoryAnnotations<TFramework, TArgs>, 'story'>;
-};
+} & ({} extends TArgsAnnotations ? { args?: TArgsAnnotations } : { args: TArgsAnnotations });
```

Comment:
Can you explain this line for me?

Reply:
It means, that if we can assign an empty object to TArgsAnnotation, then args should be optional, otherwise they are not optional.

Basically, if you specify all args in meta, you can both assign an empty object to args in a story, or leave the args out completely.